source/future.tex

```diff
@@ -1628,3 +1628,404 @@
 \begin{itemdescr}
 \pnum \returns \tcode{binary_negate<Predicate>(pred)}.
 \end{itemdescr}
+
+\rSec1[depr.default.allocator]{The default allocator}
+
+\pnum
+The following members and explicit class template specialization are defined in
+addition to those specified in \ref{default.allocator}:
+
+\indexlibrary{\idxcode{allocator}}%
+\begin{codeblock}
+namespace std {
+  // specialize for \tcode{void}:
+  template <> class allocator<void> {
+  public:
+    using value_type    = void;
+    using pointer       = void*;
+    using const_pointer = const void*;
+    // reference-to-\tcode{void} members are impossible.
+
+    template <class U> struct rebind { using other = allocator<U>; };
+  };
+
+  template <class T> class allocator {
+   public:
+    using size_type       = size_t;
+    using difference_type = ptrdiff_t;
+    using pointer         = T*;
+    using const_pointer   = const T*;
+    using reference       = T&;
+    using const_reference = const T&;
+    template <class U> struct rebind { using other = allocator<U>; };
+
+    T* address(reference x) const noexcept;
```

Comment:
That makes perfect sense and I wish I had noticed.  The wording approved by LWG has the same error, but do we agree this fix should be editorial?  We are talking about the nested 'reference' type of a specific allocator, not some generic customization point.